### PR TITLE
merge afeuersein/fosscord-client to fosscord/fosscord-client

### PR DESCRIPTION
**Pull request by afeuerstein**
- add translations for the instances page
- removed a space from the translations of login and register
- set the sidebar header and the content header of instances to the same height
- resize the buttons in `instances`to have the same height as the input field
- changed `I have read and agree to the Terms of Service` to `I agree to this instance's terms`
- added `www/` to `.gitignore`

I hope you're able to merge this, since I'm one commit behind. I don't know if I have to do anything at my end, my git knowledge isn't that big.

### CODE REVIEW

**Comment by xnacly** [comment]
Could you provide screenshots of the changed css values impact?

File: public/locales/en/login.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
 	"password": "Password",
 	"forgotPassword": "Forgot your password?",
 	"login": "Login",
-	"registerNotice": "Don't have an account? ",
+	"registerNotice": "Don't have an account?",
```
Comment (samuelscheit) [inline public/locales/en/login.json:6]: the space is needed, for the Register link button to have a space between it and the text

Comment (samuelscheit) [reply public/locales/en/login.json:6]: i'll check it out later

Comment (afeuerstein) [reply public/locales/en/login.json:6]: Nope, I fixed that. I added a space between the references to the translations.

Comment (samuelscheit) [reply public/locales/en/login.json:6]: ok good

**Comment by afeuerstein** [comment]
@xNaCly 
> Could you provide screenshots of the changed css values impact?

Before:
image
After:
image

**Comment by afeuerstein** [comment]
Okay, I think I successfully merged the two repos.

**Pull request merged**